agent/src/main/bin/testgrid-agent

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
```

Comment:
The purpose of this script is a bit unclear.

Reply:
This is using to facilitate agent as a int.d compatible Linux service